src/index.html

```diff
@@ -3,13 +3,115 @@
+    <fieldset class="inputs-block">
+      <legend>Personal information</legend>
+      <p>
+        <label for="surname">Surname:</label>
+        <input type="text" id="surname" name="surname">
+      </p>
+      <p>
+        <label for="name">Name:</label>
+        <input type="text" id="name" name="name">
+      </p>
+      <p>
+        <label for="age">How old are You?</label>
+        <input
+        type="number"
+        name="age"
+        id="age"
+        min="1"
+        max="100"
+        value="12">
+      </p>
+      <p>
+        <label for="data">Full date of birth:</label>
+        <input type="date" id="data" name="data">
+      </p>
+      <p class="last">
+        <label for="accept">I accept the term of the agreement</label>
+        <input type="checkbox" name="accept" id="accept">
+      </p>
+    </div>
+    </fieldset>
+    <fieldset class="inputs-block">
+      <legend>Registration</legend>
+      <p>
+        <label for="email">E-mail:</label>
+        <input
+        type="email"
+        id="email"
+        name="email"
+        required
+        placeholder="email@example.com"
+        minlength="10"
+        maxlength="15">
+      </p>
+      <p class="last">
+        <label for="pass">Password:</label>
+        <input type="password" id="pass" name="pass">
+      </p>
+    </fieldset>
+    <fieldset class="inputs-block">
+      <legend>An interesting fact about you!</legend>
+      <p>
+        <label>
+          Do you love cats?
+          <input type="radio" name="cats" value="yes"> Yes
+        </label>
+        <label>
+          <input type="radio" name="cats" value="no"> No
+        </label>
+      </p>
+      <p>
+        <label for="color">What is your favorite color?</label>
+        <input type="color" name="color" id="color">
+      </p>
+      <p>
+        <label for="time">What time do you go to bed?</label>
+        <input type="time" name="time" id="time">
+      </p>
+      <p>
+        <label for="cars">What are your favorite brand of cars?</label>
+        <select multiple id="cars" name="mul">
+          <option>BMW</option>
+          <option>Audi</option>
+          <option>Lada</option>
+        </select>
+      </p>
+      <p class="last">
+        <label for="range">How do you rate our work?</label>
+        <input type="range" id="range" name="range">
+      </p>
+    </fieldset>
+    <fieldset class="inputs-block">
+      <legend>Additional info</legend>
+      <p>
+        <label for="comments">Comments:</label>
+        <textarea name="comment" id="comments"></textarea>
+      </p>
+      <p class="last">
+        <label for="rec">Would you recommend us?</label>
+        <select id="rec" name="rec">
+          <option>Yes</option>
+          <option>No</option>
+        </select>
+      </p>
+    </fieldset>
+  </form>
+  <p>
+    <input
+    type="submit"
+    name="but"
+    value="Submit"
+    onsubmit="onSubmit ()">
```

Comment:
```suggestion
    onsubmit="onSubmit()">
```
no spaces between the name of the function and brackets